### ISSUE TITLE
Engine scaled wider than my monitor

### ISSUE THREAD

**Issue by Tohzt** [godotengine/godot#85424]
### Godot version

4.1

### System information

Windows 11

### Issue description

Very strange issue with my editor. I was testing a game and hit something on my bluetooth controller by mistake (Not sure if this was the cause). I don't know what happened but my editor is now way wider than my monitor so a lot or the editor is off-screen.. I don't know what to do about this. I've tried changing the size and monitor display and the editor settings and nothing helps.
I've even restarted my pc and reinstalling Godot and its still like this

![Screenshot 2023-11-27 011249](https://github.com/godotengine/godot/assets/19990350/af704fc0-3927-4723-ba4b-b166594e25ff)

### Steps to reproduce

I have no idea...
It only seems to bug out on the project I was on but I don't know of any project settings that could have done this.

### Minimal reproduction project

No idea

**Comment by Tohzt** [comment]
Someone in discord told me to delete all of the text in the project.godot file. This worked. Still zero idea how this happened...

**Comment by Tohzt** [comment]
> > Someone in discord told me to delete all of the text in the project.godot file. This worked. Still zero idea how this happened...
> 
> In the future, please post the contents of `project.godot` here before removing it.

I thought that all of that info looked fine but I guess that wasn't the case.... it happened again. This time I was changing my game window to be large and shrunk down, which was working fine, but then I tried to make it a little taller and the scaling issue came back. Hers is the project.godot content:

`; Engine configuration file.
; It's best edited using the editor UI and not directly,
; since the parameters that go here are not all obvious.
;
; Format:
;   [section] ; section goes between []
;   param=value ; assign values to parameters

config_version=5

[application]

config/name="Dungeon Strikers"
run/main_scene="res://game.tscn"
config/features=PackedStringArray("4.1")

[autoload]

Globals="*res://globals.gd"
GameManager="*res://game_manager.gd"

[display]

window/size/viewport_width=5120
window/size/viewport_height=3072
window/size/window_width_override=1024
window/size/window_height_override=614
window/stretch/mode="canvas_items"

[gui]

theme/custom="res://Scenes/UI/src/ui_theme.tres"
theme/custom_font="res://Scenes/UI/src/light_pixel-7.ttf"

[input]

LEFT={
"deadzone": 0.5,
"events": [Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194319,"key_label":0,"unicode":0,"echo":false,"script":null)
, Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":65,"key_label":0,"unicode":65,"echo":false,"script":null)
, Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":0,"axis_value":-1.0,"script":null)
]
}
RIGHT={
"deadzone": 0.5,
"events": [Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194321,"key_label":0,"unicode":0,"echo":false,"script":null)
, Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":68,"key_label":0,"unicode":68,"echo":false,"script":null)
, Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":0,"axis_value":1.0,"script":null)
]
}
UP={
"deadzone": 0.5,
"events": [Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194320,"key_label":0,"unicode":0,"echo":false,"script":null)
, Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":87,"key_label":0,"unicode":87,"echo":false,"script":null)
, Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":1,"axis_value":-1.0,"script":null)
]
}
DOWN={
"deadzone": 0.5,
"events": [Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194322,"key_label":0,"unicode":0,"echo":false,"script":null)
, Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":83,"key_label":0,"unicode":83,"echo":false,"script":null)
, Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":1,"axis_value":1.0,"script":null)
]
}
AIM_LEFT={
"deadzone": 0.5,
"events": [Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":2,"axis_value":-1.0,"script":null)
]
}
AIM_RIGHT={
"deadzone": 0.5,
"events": [Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":2,"axis_value":1.0,"script":null)
]
}
AIM_UP={
"deadzone": 0.5,
"events": [Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":3,"axis_value":-1.0,"script":null)
]
}
AIM_DOWN={
"deadzone": 0.5,
"events": [Object(InputEventJoypadMotion,"resource_local_to_scene":false,"resource_name":"","device":-1,"axis":3,"axis_value":1.0,"script":null)
]
}
Attack={
"deadzone": 0.5,
"events": [Object(InputEventMouseButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"button_mask":0,"position":Vector2(0, 0),"global_position":Vector2(0, 0),"factor":1.0,"button_index":1,"canceled":false,"pressed":false,"double_click":false,"script":null)
]
}
`

**Comment by Duskitten** [comment]
Am working on a project with a team, and we've noticed this issue comes into play when we set the custom theme, as it appears you have this as well, so perhaps its an issue with the custom theme loader interacting with the editor for some reason?

**Comment by Tohzt** [comment]
> Am working on a project with a team, and we've noticed this issue comes into play when we set the custom theme for the project, as it appears you have this as well, so perhaps its an issue with the custom theme loader interacting with the editor for some reason?

Indeed, I noticed last night, through process of elimination, that this is somehow tied to the custom project theme. If I use the theme individually on my components, everything is fine.

**Comment by YuriSizov** [comment]
Please upload an MRP with a theme resource that you think causes this issue.

**Comment by Tohzt** [comment]
> Please upload an MRP with a theme resource that you think causes this issue.

I'm not really sure how to reproduce this. I've made themes in similar ways before and have never seen this issue. I was also using this theme for a while before the issue presented itself and only later realized that removing the theme from the project settings seemed to fix it.

**Comment by YuriSizov** [comment]
I'm not sure about the editor getting too wide, but the issue with the scene tabs getting so tall, as demonstrated on the screenshot in the OP, was reported before and the cause of it should be fixed in 4.2.

Can you somehow test your setup there?

**Comment by Tohzt** [comment]
> I'm not sure about the editor getting too wide, but the issue with the scene tabs getting so tall, as demonstrated on the screenshot in the OP, was reported before and the cause of it should be fixed in 4.2.
> 
> Can you somehow test your setup there?

I've been meaning to give 4.2 a try. Should have some time this weekend
